ansible/roles/cartodb.nodejs/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+ - name: Add cartodb nodejs ppa
+   apt_repository: repo=ppa:cartodb/nodejs-010 state=present update_cache=yes
+
+ - name: Install nodejs 0.10.x
+   apt: name=nodejs state=present
```

Comment:
Hmmm the version concerns are valid. I'm not sure how this will shake out over time, although I expect cartodb to add new repos when they update versions, rather than updating the package in the repo, since the package version is in the repository name. I'm inclined to leave this as is for now and re-evaluate later if cartodb starts making significant changes to these repos later.